include/pqxx/result.hxx

```diff
@@ -194,6 +194,20 @@ public:
   /// Name of column with this number (throws exception if it doesn't exist)
   [[nodiscard]] char const *column_name(row_size_type number) const &;
 
+  /// Size in bytes of column with this number
```

Comment:
Thanks for documenting!  I have some things to add, but don't worry about it — I can do that after merging.